english/you-are-in-minecraft/gather_wood/tame_the_elephant/tame_the_elephant.md

```diff
@@ -1,2 +1,4 @@
 You know you always wanted to have a pet elephant in minecraft, so you can kill all those zombies with a single whip.
-You decide to tame it.
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.